packages/nextjs/app/builders/0x21dfd1CfD1d45801f46B0F40Aed056b064045aA2/page.tsx

```diff
@@ -0,0 +1,147 @@
+"use client";
```

Comment:
You don't need to use "use client" here, as this can be server-rendered. There are no client-side actions like useEffect or useState involved.